src/main/java/com/jherrick/ConsolePlayerAgent.java

```diff
@@ -3,12 +3,17 @@
 import java.util.Scanner;
 
 public class ConsolePlayerAgent implements Agent {
+    
     private final Scanner reader;
 
+
     public ConsolePlayerAgent(Scanner reader) {
         this.reader = new Scanner(System.in);;
     }
 
+
```

Comment:
Just go through the code changes on this page and remove the big chunks of whitespace. Everything should pretty much just be separated by a single line - at most 2 for a significant break.